api/ccapi/include/model.h

```diff
@@ -40,6 +41,21 @@ enum class ModelType {
   UNKNOWN     /** Unknown */
 };
 
+/**
+ * @brief Model saving options
+ *
+ */
+enum class ModelFormat {
+  MODEL_FORMAT_BIN =
+    ML_TRAIN_MODEL_FORMAT_BIN, /**< raw bin file saves parameters required
+for inference and training without any configurations*/
+
```

Comment:
at this time, we only save and load updated weights. ( there is only one format we support for now, FORMAT_BIN ).
That means, we are not update the ini. and if it is true, then the model save is always the same between training and infernece.

Reply:
Yes :) I think we will have `ML_TRAIN_MODEL_FORMAT_OPTIMIZED_FOR_INFERENCE`  this kind of enum.